lib/segregate.rb

```diff
@@ -236,14 +251,27 @@ def parse_body line
     @stashed_body = ""
 
     if line.length >= headers['content-length'].to_i
-      @body = line
+      line = line[0..-3]
```

Comment:
I am removing the two delimiters characters from the HTTP protocol. I can add a comment to be clear.